Mon Jun 12 03:28:11 CEST 2006  VMiklos <vmiklos@frugalware.org>
  * makepkg: don't halt on missing install scriplet
  it can be ok for example when the scriptlet will be created during build()
  (inheritation from scheme)
diff -rN -u old-pacman/scripts/makepkg new-pacman/scripts/makepkg
--- old-pacman/scripts/makepkg	2006-06-12 03:32:42.000000000 +0200
+++ new-pacman/scripts/makepkg	2006-06-12 03:32:42.000000000 +0200
@@ -735,8 +735,7 @@
 	exit 1
 fi
 if [ "$install" -a ! -f "$install" ]; then
-	error "install scriptlet ($install) does not exist."
-	exit 1
+	warning "install scriptlet ($install) does not exist."
 fi
 
 if [ -f $PKGDEST/${pkgname}-${pkgver}-${pkgrel}-${CARCH}.fpm -a "$FORCE" = "0" -a "$GENMD5" = "0" -a "$GENSHA1" = "0" ]; then

